CONTRIBUTING.md

```diff
@@ -41,38 +41,24 @@ steps:
 so that you have a copy on GitHub that you can clone and work with
 3) Clone the codebase down to your local machine
 4) For the code to run locally without getting a `ModuleNotFoundError`,
-you need to remove the import prefix `git_sim.` from imports (for example
-`from git_sim_base_command import GitSimBaseCommand`) in the following files:
-
-- __main.py__
-- git_sim.py
-- git_sim_add.py
-- git_sim_branch.py
-- git_sim_cherrypick.py
-- git_sim_commit.py
-- git_sim_log.py
-- git_sim_merge.py
-- git_sim_rebase.py
-- git_sim_reset.py
-- git_sim_restore.py
-- git_sim_revert.py
-- git_sim_stash.py
-- git_sim_status.py
-- git_sim_tag.py
-
-There is probably a way around this. When I figure it out I'll update this guide.
-
-5) You can run your local Git-Sim commands from within other local repos like this:
-
+install the developement package by running
+```console
+$ cd git-sim
+$ python setup.py develop
```

Comment:
```suggestion
$ python -m pip install --no-deps -e .
```
Or perhaps you want the deps.

Caution that directly invoking `setup.py` is now deprecated.
* https://pip.pypa.io/en/stable/reference/build-system/setup-py (legacy)
* https://pip.pypa.io/en/stable/topics/secure-installs/?highlight=setup.py#do-not-use-setuptools-directly